packages/side-nav/src/containers/Docked.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+import { ThemeContext } from "@hig/themes";
+
+import "./docked.scss";
+
+export default class Docked extends Component {
```

Comment:
This Docked container keeps the SideNav to the left of all content, including the TopNav. I intend to write another to fit the SideNav below the TopNav.

Reply:
If we want to reserve the `containers` name for encapsulating state, maybe we can come up with a better name. `Wrappers`, `Positioners`, `Orientations` 🤷‍♂️

Reply:
My first question is, does this even belong in the side-nav package? This affects the entire screen, which is a concern for layouts. However, I'd like to digest that thought a bit more, so leave it alone for now.

Reply:
Great points. One step at a time. Will keep that in mind / note in code comments later.